jetstream/statistics.py

```diff
@@ -422,6 +423,44 @@ def transform(
         )
 
 
+@attr.s(auto_attribs=True)
+class LinearModelMean(Statistic):
+    drop_highest: float = 0.005
+    covariate_adjustment: dict[str, dict[str, str]] | None = None
```

Comment:
Right, these parameters are coming straight from metric-hub. 

I decided to nest them for organization, so the config could look like: 

```toml
[metrics]
[metrics.my_metric]
...
[metrics.my_metric.statistics.linear_model_mean]
drop_highest = 0
[metrics.my_metric.statistics.linear_model_mean.covariate_adjustment]
metric = "metric"
period = "period"
```

One alternative to nesting could be to put everything at the top level, maybe prefixed. Something like: 

```toml
[metrics]
[metrics.my_metric]
...
[metrics.my_metric.statistics.linear_model_mean]
drop_highest = 0
covariate_adjustment_metric = "metric"
covariate_adjustment_period = "period"
```

This would make the typing clearer in the `Statistic`, but then future parameters all need that prefix. Thoughts?

Reply:
Eh I think what you have is fine. I was looking at it in a vacuum without thinking about where it was coming from. If you want to be clearer in the class, you could add a comment about what you expect to be in the dict? Only problem is that could get out of date, so maybe not worth it.

Reply:
Plus what you have is more consistent with current statistic parameters, so yea another vote in favor of what you have now.

Reply:
Great, then I'll leave as-is